ipa-core/src/protocol/context/mod.rs

```diff
@@ -1,9 +1,13 @@
+#[allow(dead_code)]
+mod dzkp_malicious;
```

Comment:
you would make it better if you add a folder zkp, put all implementations there and strip DZKP prefix from everywhere: `context::dzkp_validator::DZKPValidator` really does not need to say it twice

Reply:
Yes, I think that would be much better. Unfortunately as mentioned above there are parts of the protocol where we want to use MACs and dzkps. I added the prefix since otherwise there are many naming collisions with the MAC based approach. We could strip it and then renaming it when importing the module if you prefer that. I find the renaming (as in case of the additive/replicated shares) more confusing.